test/functional/update/slow-server/server.py

```diff
@@ -15,21 +16,32 @@ def do_GET(self):
             self.end_headers()
             return
 
-        self.send_response(200)
+        #Simulate partial download
+        partial_download = False
+        if self.path == "//100/pack-os-core-from-10.tar" and SlowResponse.first_time:
+            partial_download = True
+            SlowResponse.first_time = False
+            self.send_response(206)
+        else:
+            self.send_response(200)
         self.send_header("Content-type", "text/html")
         self.end_headers()
         delay = 0.00001 # seconds
 
         while True:
-            b = f.read(1000)
+            b = f.read(100)
             if b == b'':
                 break
+
             self.wfile.write(b)
+            if partial_download:
+                break;
             time.sleep(delay)
         self.wfile.flush()
         f.close()
 
 if __name__ == '__main__':
+    first_time = True
```

Comment:
Why a constant here and a class-level constant in SlowResponse?
My google-fu is failing me in trying to remember, and now my curiosity is getting to me :smile:

Reply:
oops. This should be removed